sxt/field12/type/element.h

```diff
@@ -48,6 +49,8 @@ class element {
 
   CUDA_CALLABLE constexpr uint64_t* data() noexcept { return data_; }
 
+  static constexpr element modulus() noexcept { return f12b::p_v.data(); }
```

Comment:
Please change the array constructor for element to be explicit

Reply:
Good idea. Updated.